src/screens/App/Assets/Icons/circle-graph.svg

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Nothing wrong with the svg, but typically you want to name your folders all lowercased (I noticed this with the repo name itself but didn't want to nitpick). I'm not good at explaining conventions I learned to follow so I found this explanation online at [MDN](https://developer.mozilla.org/en-US/docs/Learn/Getting_started_with_the_web/Dealing_with_files)

Reply:
Yep, I can change it - I actually prefer lowercase, but I thought uppercase was more correct